notebooks/book2/11/mcmc_gmm_demo.ipynb

```diff
@@ -1,9 +1,17 @@
 {
```

Comment:
<p>Given the repeated code, can we use a function or a loop to avoid code duplication?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>